copulas/univariate/gaussian_kde.py

```diff
@@ -180,6 +181,45 @@ def percent_point(self, U):
 
         return X
 
+    def percent_point_fast(self, U):
+        """Compute the inverse cumulative distribution value for each point in U.
+
+        Arguments:
+            U (numpy.ndarray):
+                Values for which the cumulative distribution will be computed.
+                It must have shape (n, 1) and values must be in [0,1].
+
+        Returns:
+            numpy.ndarray:
+                Inverse cumulative distribution values for points in U.
+
+        Raises:
+            NotFittedError:
+                if the model is not fitted.
+        """
+        self.check_fit()
+
+        assert len(U.shape) == 1
+
+        if np.any(U > 1.0) or np.any(U < 0.0):
+            raise ValueError("Expected values in range [0.0, 1.0].")
+
+        is_one = U >= 1.0 - EPSILON
+        is_zero = U <= EPSILON
+        is_valid = np.logical_not(np.logical_or(is_zero, is_one))
+
+        lower, upper = self._get_bounds()
+
+        def _f(X):
+            return U[is_valid] - self.cumulative_distribution(X)
+
+        X = np.zeros(U.shape)
+        X[is_one] = float("inf")
+        X[is_zero] = float("-inf")
+        X[is_valid] = newton(_f, np.zeros(U[is_valid].shape) + (lower + upper) / 2.0)
```

Comment:
You could use `np.full` instead of doing `np.zeros + value`, right?

Reply:
yep